docs/components/SideBar/SideBar.jsx

```diff
@@ -20,6 +20,7 @@ export default function SideBar({className}) {
         <NavLink href="/design/typography">Typography</NavLink>
         <NavLink href="/design/sizing">Sizing</NavLink>
         <NavLink href="/design/page-layout">Page Layout</NavLink>
+        <NavLink href="/design/compound-form">Compound Form</NavLink>
```

Comment:
I prefer alphabetical when I'm trying to get an overview. But I guess this is out of scope for this PR. Thanks for the clarification.